src/Sonarr.Http/REST/RestController.cs

```diff
@@ -87,7 +88,8 @@ public override void OnActionExecuting(ActionExecutingContext context)
                 }
             }
 
-            var attributes = descriptor.MethodInfo.CustomAttributes;
+            var attributes = descriptor.MethodInfo.CustomAttributes as IReadOnlyCollection<CustomAttributeData> ??
```

Comment:
Backing type is ICollection (but maybe depends on implementation), so this change mostly just suppresses the warning